kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.11 on 2024-07-31 20:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('audit_log', '0001_squashed_0006_remove_index_together'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='auditlog',
+            name='action',
+            field=models.CharField(choices=[('create', 'CREATE'), ('delete', 'DELETE'), ('in-trash', 'IN TRASH'), ('put-back', 'PUT BACK'), ('remove', 'REMOVE'), ('update', 'UPDATE'), ('auth', 'AUTH')], db_index=True, default='delete', max_length=10),
+        ),
+        migrations.AddIndex(
+            model_name='auditlog',
+            index=models.Index(models.F('metadata__asset_uid'), models.F('action'), name='audit_log_asset_action_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='auditlog',
+            index=models.Index(models.F('metadata__asset_uid'), name='audit_log_asset_uid_idx'),
+        ),
+    ]
```

Comment:
We should use `SKIP_HEAVY_MIGRATIONS` with manual instructions for big databases. 
Index creation could lock the table for a while and would make the app unresponsive. 

Have a look at https://github.com/kobotoolbox/kpi/blob/main/kpi/migrations/0046_project_view_assets_indexes.py for an example.

Reply:
Do we expect the audit log tables to already be large? i thought they only had deletions in them right now

Reply:
updated in any case

Reply:
Yes, it already has a lots of records.